py/torch_tensorrt/dynamo/_compiler.py

```diff
@@ -341,4 +429,6 @@ def compile_module(
     if fast_partitioner_failed:
         settings.use_fast_partitioner = True
 
+    dryrun_stats_display(dryrun_tracker, settings.dryrun)
```

Comment:
Pass in graph including the Torch module/node information for proper display formatting